internal/cli/install.go

```diff
@@ -60,8 +60,17 @@ func (c *InstallCommand) Run(args []string) int {
 
 	p, ok := serverinstall.Platforms[strings.ToLower(c.platform)]
 	if !ok {
+		if c.platform == "" {
+			c.ui.Output(
+				"The -platform flag is required.",
+				terminal.WithErrorStyle(),
+			)
+
+			return 1
+		}
+
 		c.ui.Output(
-			"Error installing server into %s: invalid platform",
+			"Error installing server into %q: invalid platform",
```

Comment:
This was a bit of feedback on my upgrade PR, so I think it makes sense to do this here too:
```suggestion
			"Error installing server into %q: unsupported platform",
```

Reply:
I _swear_ i put the `%q` in there! Must have dropped it amidst some stashes 😄 Thanks!

Reply:
Ignore my previous comment; the git suggestion diff confused me 😄